hive-tables-creation.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+
+client = boto3.client('glue')
+response = client.create_table(
+    DatabaseName='analytical_dataset_generation',
+    TableInput={
+        'Name': 'core_contract_adg',
+        'Description': 'core_contract_adg',
+        'StorageDescriptor': {
+            'Columns': [
+                {
+                    'Name': 'rowkey',
+                    'Type': 'string'
+                },
+                {
+                    'Name': 'data',
+                    'Type': 'string'
+                }
+            ],
+            'Location': 's3://{REPLACE_THIS}/hive/external/core_contract_adg',
+            'Compressed': False,
+            'NumberOfBuckets': -1,
+            'SerdeInfo': {
+                'Name': 'string',
+                'SerializationLibrary': 'org.apache.hadoop.hive.hbase.HBaseSerDe',
+                'Parameters': {
+                    'hbase.columns.mapping': ':key,cf:record',
+                    'serialization.format': '1'
+                }
+            },
+        },
+        'TableType': 'EXTERNAL_TABLE',
+        'Parameters': {
```

Comment:
I don't understand what these parameters are? Why are we referencing HBase tables here if we're creating external tables over S3? Can the whole `Parameters` block just be removed?

Reply:
Data access pattern we followed to access hbase is by creating hive table over it as it can be performant due to full table scan . Approached as per https://diogoalexandrefranco.github.io/interacting-with-hbase-from-pyspark/